docs/source/definition-of-done.md

```diff
@@ -0,0 +1,12 @@
+# Definiton of done
+
+ * [ ] The acceptance criteria described in the Task is met
+ * [ ] The code is approved and merged into the repository(ies)
+ * [ ] The documentation is updated
+   * Official landing page: [](https://leapp.readthedocs.io/en/latest/contributing.html)
+   * Developers documentation: https://leapp.readthedocs.io/en/latest/
+   * Presentation for external contributors: https://goo.gl/Foh2tk
+ * [ ] Dev. Images are updated if necessary
+ * [ ] Unit tests that cover the new code are written and passing
+ * [ ] Integration tests are passing
+ * [ ] The ticket contains link(s) with the results of the Task (e.g., PR, document, etc.).
```

Comment:
```suggestion
 * [ ] The Jira Issue contains link(s) to the results (e.g., PR, document, etc.).
```

Reply:
waiting for a decision regarding existency of definition of done